cli/tokens.css

```
@@ -0,0 +1,137 @@
+  --tool-accent: var(--color-orange);
+  --tool-surface: #2a2218;
+  --tool-contrast: var(--color-ink-900);
+
+  /* Per-racer accent: deliberately NOT declared here. It is set inline as
+     `style="--racer-color: #e74c3c"` on the element that needs it; while it is
+     unset every `var(--racer-color, <neutral>)` below resolves to its neutral
+     fallback. Declaring it at :root would only mask that. */
+
+  /* --- Typography ------------------------------------------------------- */
+  --font-ui: ui-monospace, 'Courier New', monospace;
+  --font-display: Georgia, 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)